src/anomalib/metrics/per_image/pimo_numpy.py

```diff
@@ -102,11 +100,10 @@ def pimo_curves(
             [3] image classes of shape (N,) with values 0 (normal) or 1 (anomalous)
     """
     # validate the strings are valid
-    BinclfAlgorithm(binclf_algorithm)
     _validate.is_num_threshs_gte2(num_threshs)
-    _validate.is_anomaly_maps(anomaly_maps)
-    _validate.is_masks(masks)
-    _validate.is_same_shape(anomaly_maps, masks)
+    _validate.is_anomaly_maps(anomaly_maps)  # redundant
+    _validate.is_masks(masks)  # redundant
+    _validate.is_same_shape(anomaly_maps, masks)  # redundant
```

Comment:
they are redundant with the `PIMO` class, right?
but if you call it without the class they aren't

Reply:
These were just for myself. I am going to refactor the validation stuff so that we do it once at the outermost level.

Reply:
It would be good to have @jpcbertoldo as a reviewer in your pr series..

Reply:
i am checking them ;)